### PR TITLE
🐛 General update of chain models

### PR DESCRIPTION
**Pull request by jakvbs**
Closes https://github.com/TrueFiEng/useDApp/issues/994

### CODE REVIEW

**Review by yivlad — APPROVED** [review]

File: packages/core/src/model/chain/metis.ts [hunk 1]
```diff
@@ -1,23 +1,42 @@
...
 
 export const Stardust: Chain = {
   chainId: 588,
   chainName: 'Stardust',
   isTestChain: true,
   isLocalChain: false,
   multicallAddress: '0xaF9D4DC0698d8FD9f41387ecb08D9976079B8086',
-  getExplorerAddressLink: (address: string) => `https://stardust-explorer.metis.io/address/${address}`,
-  getExplorerTransactionLink: (transactionHash: string) => `https://stardust-explorer.metis.io/tx/${transactionHash}`,
+  rpcUrl: 'https://stardust.metis.io/?owner=588',
```
Comment (rzadp) [inline packages/core/src/model/chain/metis.ts:12]: Who is the owner? Does it have any significance?

File: packages/core/src/model/chain/metis.ts [hunk 2]
```diff
@@ -1,23 +1,42 @@
...
+
 export const Andromeda: Chain = {
   chainId: 1088,
   chainName: 'Andromeda',
   isTestChain: false,
   isLocalChain: false,
   multicallAddress: '0x1a2AFb22B8A90A77a93e80ceA61f89D04e05b796',
-  getExplorerAddressLink: (address: string) => `https://andromeda-explorer.metis.io/address/${address}`,
-  getExplorerTransactionLink: (transactionHash: string) => `https://andromeda-explorer.metis.io/tx/${transactionHash}`,
+  rpcUrl: 'https://andromeda.metis.io/?owner=1088',
```
Comment (rzadp) [inline packages/core/src/model/chain/metis.ts:31]: Who is the owner? Does it have any significance?

File: packages/core/src/model/chain/palm.ts [hunk 3]
```diff
@@ -1,23 +1,42 @@
...
 
 export const Palm: Chain = {
   chainId: 11297108109,
   chainName: 'Palm',
   isTestChain: false,
   isLocalChain: false,
   multicallAddress: '0x99a73dfE34578348fb81BD078201C0BA84E9c840',
-  getExplorerAddressLink: (address: string) => `https://explorer.palm.io/address/${address}`,
-  getExplorerTransactionLink: (transactionHash: string) => `https://explorer.palm.io/tx/${transactionHash}`,
+  rpcUrl: 'https://palm-mainnet.infura.io/v3/3a961d6501e54add9a41aa53f15de99b',
```
Comment (rzadp) [inline packages/core/src/model/chain/palm.ts:12]: Whose id is this?
If there is no public rpc without IDs, we should maybe skip the `rpcUrl` here

**Comment by hiromaily** [comment]
@jakvbs Thanks for your great help!

**Pull request merged**